src/main/java/Event.java

```diff
@@ -0,0 +1,16 @@
+public class Event extends Task {
+
+    public String from;
+    public String to;
```

Comment:
Should "to" and "from" be private or protected, since they should be encapsulated and hidden from the user unless through getter methods?